src/status_im/ui/screens/db.cljs

```diff
@@ -176,6 +177,10 @@
 
 (spec/def ::device-UUID (spec/nilable string?))
 
+;;;; Supported Biometric authentication types
+
+(spec/def ::supported-biometric-auth (spec/nilable keyword?))
```

Comment:
is it just one single keyword? how does it work? is the list open or could we have a set of possible keywords here? I don't find any exemple of what I should expect here

Reply:
@yenda yes it is a keyword only, inherited by touch-id module.
`:TouchID` (ios)
`:FaceID` (ios)
`:fingerprint` (android)
`nil` (not supported\desktop)

I can make it a strict set of possible keywords.

Reply:
@yenda done